### DISCUSSION TITLE
新版本带上openAI的key无法使用

### DISCUSSION THREAD

**Discussion by qzdemon** [lobehub/lobehub#2217] [Self Hosting] [answered]
新版本带上openAI的key启动docker，进入页面后交互对话还是提示需要填写key

**Comment by arvinxx** [comment]
请给出更完整的信息

**Reply by qzdemon** [reply]
我在封装启动docker的时候根据官方的命令  -e AZURE_API_KEY= 带上了key  但是进入页面交互对话显示  openAI key为空，请添加自定义openAI API key

**Reply by qzdemon** [reply]
也带上了 -e OPENAI_PROXY_URL=

**Reply by arvinxx** [reply]
Azure 看下https://lobehub.com/zh/docs/self-hosting/examples/azure-openai

**Reply by qzdemon** [reply]
我使用您更新后的Azure的集成方式  使用AZURE_API_KEY、AZURE_ENDPOINT、AZURE_MODEL_LIST、AZURE_API_VERSION可是还是一样的问题

**Reply by arvinxx** [reply, marked as answer]
可能要手动选一下模型。或者配置下 `DEFAULT_AGENT_CONFIG` 这个变量： https://lobehub.com/zh/docs/self-hosting/environment-variables/basic#default-agent-config 。默认的 provider 是 openai ，你可能要换成 `provider=azure`
